src/Test/Perf/util.csx

```diff
@@ -0,0 +1,180 @@
+#r "..\..\..\Binaries\Debug\Microsoft.CodeAnalysis.Scripting.dll"
+#r "..\..\..\Binaries\Debug\Microsoft.CodeAnalysis.CSharp.Scripting.dll"
```

Comment:
Yeah...  The good news is that since this is a csx script, these DLLs do have to be on the system.  But yeah, right now this depends on a local build of those Dlls.